python/dolfinx/io.py

```diff
@@ -15,10 +15,55 @@
 from dolfinx import cpp as _cpp
 from dolfinx.cpp.io import distribute_entity_data
 from dolfinx.cpp.io import perm_gmsh as cell_perm_gmsh
+from dolfinx.common import has_adios2
```

Comment:
I don't think we this. Python way is try/except rather than checking that ADIOS2 is availably.